### PR TITLE
Migrate CI to GitHub Actions

### PR DESCRIPTION
**Pull request by tsibley**
We've been slowly moving everything off Travis CI in favor of
centralizing on GitHub.  More acutely motivated here by some
Travis-specific errors, previously worked around by
<https://github.com/nextstrain/zika/pull/14>.

Reverts back to a simple pip install instead of conda/mamba setup as
much of it was unnecessary.  See discussion in
<https://github.com/nextstrain/measles/pull/3#discussion_r840793744>.

### CODE REVIEW

**Comment by tsibley** [comment]
If this is acceptable to folks, then I would also plan to refactor it out into https://github.com/nextstrain/.github as a [starter workflow](https://docs.github.com/en/actions/using-workflows/creating-starter-workflows-for-your-organization) + [reusable workflow](https://docs.github.com/en/actions/using-workflows/reusing-workflows) pair that we can then apply (and maintain) across the board to all/most of our pathogen repos.

**Review by j23414 — APPROVED** [review]
It passed travis checks, and I like the migration to GH so looks good to me! Thanks!

File: README.md [hunk 1]
```diff
@@ -13,8 +13,6 @@ housed in this repository.
 
 _This build requires Augur v6._
 
-[![Build Status](https://travis-ci.com/nextstrain/zika.svg?branch=master)](https://travis-ci.com/nextstrain/zika)
```
Comment (victorlin) [inline README.md:16]: Replace with GitHub's equivalent?

```
[![Build Status](https://github.com/nextstrain/zika/actions/workflows/ci.yaml/badge.svg?branch=master)](https://github.com/nextstrain/zika/actions/workflows/ci.yaml)
```

**Review by victorlin — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (tsibley) [reply README.md:16]: Done.

**Review by victorlin — APPROVED** [review]

**Pull request merged**